css-shared-element-transitions/index.bs

```diff
@@ -593,254 +655,307 @@ document's DOM structure.
+            then skip painting |descendant|.
+
+            Note: This is necessary since the descendant will generate its own snapshot which will be displayed and animated independently.
+
+    1. Let |interestRectangle| be the result of [=computing the interest rectangle=] for |element|.
+
+        Note: The |interestRectangle| is the subset of |element|'s [=ink overflow rectangle=] that should be captured.
+            This is required for cases where an element's ink overflow rectangle needs to be clipped because of hardware constraints.
+            For example, if it exceeds the maximum texture size.
+
+    1. Return the portion of the canvas within |interestRectangle| as an image.
+        The natural size of the image is equal to the |interestRectangle| bounds.
 </div>
-	
+
 <div algorithm>
-	To <dfn>update transition DOM</dfn> given a {{SameDocumentTransition}} |manager|:
-	
-	1. For each [=page-transition pseudo-elements=] associated with |manager|,
-		check whether there is an active animation associated with this pseudo-element.
-	
-		Issue: Define what active animation means here.
-	
-	1. If no [=page-transition pseudo-elements=] has an active animation:
-	
-        1. Set |manager|&apos;s {{[[Phase]]}} internal slot to "idle".
-
-        1. [=Resolve=] |manager|&apos;s{{[[finished]]}} with |finishedP|.
+    To <dfn>update transition DOM</dfn> given a {{SameDocumentTransition}} |transition|:
+
+    1. For each [=page-transition pseudo-elements=] associated with |transition|,
+        check whether there is an active animation associated with this pseudo-element.
+
+        Issue: Define what active animation means here.
+
+    1. If no [=page-transition pseudo-elements=] has an active animation:
+
+        1. Set |transition|'s [=SameDocumentTransition/phase=] to "`finished`".
+
+        1. [=Resolve=] |transition|'s[=SameDocumentTransition/finished promise=].
 
         1. Return.
-	
-	1. [=map/For each=] |tag| -> |CapturedElement| of |manager|&apos;s {{[[TaggedElements]]}}:
-	
-        1. If |CapturedElement| has an "incoming element", run [=capture the image=]
-            on |CapturedElement|&apos;s "incoming element" and update the displayed
+
+    1. [=map/For each=] |tag| -> |capturedElement| of |transition|'s [=SameDocumentTransition/tagged elements=]:
+
+        1. If |capturedElement| has an "incoming element", run [=capture the image=]
+            on |capturedElement|'s "incoming element" and update the displayed
             image for ''::page-transition-incoming-image'' with the tag |tag|.
 
             At the [=user-agent origin=],
-            set |incoming|&apos;s 'object-view-box' property
+            set |incoming|'s 'object-view-box' property
             to a value that when applied to |incoming|,
             will cause the view box to coincide with "incoming element"'s [=border box=]
             in the image.
 
```

Comment:
We need to clarify that the incoming elements are not painted in the author DOM when the transition is "running".  The text could be as follows but I'm not sure is this is the spot to add it in the spec. If you're also unsure then perhaps leave an Issue to move it to the correct place.

_The "incoming element" and its contents (the [flat tree](https://drafts.csswg.org/css-scoping-1/#flat-tree) descendants of the element, including both text and elements, or the replaced content of a [replaced element](https://www.w3.org/TR/css-display-3/#replaced-element)) are not painted (as if they had [visibility: hidden](https://www.w3.org/TR/CSS2/visufx.html#propdef-visibility)) and do not respond to hit-testing (as if they had [pointer-events: none](https://www.w3.org/TR/SVG2/interact.html#PointerEventsProperty))._

Reply:
I'll add a note to figure this out